class-00/lab-c/lab/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>Choco Pizza</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles/layout.css">

```

Comment:
This looks like it could be the issue why your code is breaking. You have "styles/layout.css" and there is no "styles" directory in your repo.